test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/EnumFilterFunctionalTests.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.OData.UriParser;
 using Microsoft.OData.Edm;
 using Xunit;
+using System.Linq;

```

Comment:
`System.X` namespaces should always be at the top.

Reply:
@KenitoInc : This has been moved to the top